src/pages/en/community/nsfw-policy.mdx

```diff
@@ -0,0 +1,206 @@
+
+Generally speaking:
+
+- **`NSFW` includes:** Suggestive content, porn, etc
+- **`NSFL` includes:** Animal abuse, gore, vore, suicide, serious injuries, body horror, etc
+
+Additionally, SFW ("Safe For Work") and SFL ("Safe For Life") can be used as opposing designations for content that definitely doesn't come under the designations of NSFW or NSFL respectively.
+
+NSFL content is (and always will be) roundly banned at Quilt, aside from users seeking support from each other regarding how they feel about it, or for things that have happened to them. With NSFW content, though, the devil's in the details -- this term is loose by design, and they way it's applied differs greatly depending on the people, situation, and content in question.
+
+## Quilt's Definitions
+
+In order to make this easier to understand, we've split potentially NSFW content into several categories:
+
+- **Explicitly Allowed**: Content which we do not consider NSFW, and that we allow in any space
+- **Contextual**: Content which we may allow or disallow based on the discussion it's part of and context around it
+- **Restricted**: Content which is only allowed within the spaces we specifically designate for it
+- **Banned**: Content which is not allowed in all circumstances
+
+Some of our spaces may also have specific rules that apply to them. Where that's the case, we've listed them under the appropriate heading below.
+
+**Note:** It's always worth remembering that everyone has their own experiences and history. Additionally, note that _posting_ content and _discussing_ it isn't the same thing -- but if you're in doubt, you can always contact us via **@ModMail**.
+
+**Additionally:** We are not the ultimate arbiters of what should and should not be considered NSFW content on a global scale. Some types of content we've listed below are included because a user had questions about it, because we want to ensure people understand how posting specific content impacts their safety, or because we wanted to be clear and avoid confusing users -- not because we consider it problematic or we think other people shouldn't be allowed to peruse it, aside from a few specific examples in the banned category.
+
+<Foldable id="space-specific rules">
+<span slot="title">Space-Specific Rules</span>
+
+The following rules apply to specific Quilt community spaces. If a space isn't listed here, you can assume that it follows the rest of this document. **Banned content remains banned regardless of space-specific rules.**
+
+- **Forums:** Minimal contextual content outside designated spaces
+- **GitHub:** No contextual or restricted content
+- **Toolchain Discord:** No restricted content and minimal contextual content
+</Foldable>
+
+<Foldable id="explicitly-allowed-content">
+<span slot="title">Explicitly Allowed Content</span>
```

Comment:
Well, I guess this issue might be exaggerated and the meaning is clear for people who continue reading or re-read sections, but my point still stands, it's always nice to make things as easy to understand as possible 😁

Reply:
This was added to clarify that "Allowed Content" meant "we allow these things some people wouldn't" rather than "these are the only things we allow".

I don't think it's confusing to say "explicitly allowed content" here. This isn't "allowed explicit content". We can always clarify when people ask, b/c the uncharitable reading doesn't actually make sense in this context!

Reply:
I've had a chat on the DMD and been advised on how to approach this, so I've updated the document accordingly.

Reply:
As this is a minor wording issue, I think it's reasonable that we don't delay the new policy b/c of it - the wording can be updated at any point as needed

Reply:
> This isn't "allowed explicit content"

I've seen changes in meaning due to word order be confusing to one person before (I believe they were a russian native speaker)

> This was added to clarify that "Allowed Content" meant "we allow these things some people wouldn't" rather than "these are the only things we allow".

> We can always clarify when people ask, b/c the uncharitable reading doesn't actually make sense in this context!

Good point :+1: